app/decorators/cd_api/defence_counsel_decorator.rb

```diff
@@ -29,5 +34,22 @@ def formatted_name
     def formatted_status
       status || t('generic.not_available').downcase
     end
+
+    def formatted_defendant_names
+      names = []
+      defendants.map do |defendant|
+        names << format_defendant_name(defendant&.first_name, defendant&.middle_name, defendant&.last_name)
+      end
+
+      names.compact
+    end
+
+    def format_defendant_name(first_name, middle_name, last_name)
+      # TODO: Create a name service to build the name for reusability
```

Comment:
Rather than adding a TODO in the code, can we add a ticket in the backlog for this please

Reply:
Added a ticket, will fill out the details after planning today https://dsdmoj.atlassian.net/browse/AAC-808